CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 * [#2256](https://github.com/ruby-grape/grape/pull/2256): Raise `Grape::Exceptions::MultipartPartLimitError` from Rack when too many files are uploaded - [@bschmeck](https://github.com/bschmeck).
 * [#2266](https://github.com/ruby-grape/grape/pull/2266): Fix code coverage - [@duffn](https://github.com/duffn).
 * [#2284](https://github.com/ruby-grape/grape/pull/2284): Fix an unexpected backtick - [@zysend](https://github.com/zysend).
+* [#2287](https://github.com/ruby-grape/grape/pull/2287): Fix evaluate given in array - [@zysend](https://github.com/zysend).
```

Comment:
Since we added evaluate_given in this release, add the PR number above, so 

```
[#2285](https://github.com/ruby-grape/grape/pull/2285), [#2287](https://github.com/ruby-grape/grape/pull/2287): Added :evaluate_given to declared(params) - [@zysend](https://github.com/zysend).
```